src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ElasticPoolCrudTests.ps1

```diff
@@ -171,9 +171,10 @@ function Test-GetElasticPoolMetric
     Assert-NotNull $ep1
     
     # Get pool metrics with all values
-    $metrics = $ep1 | Get-AzureRmMetric -TimeGrain "0:5:0" -StartTime "2015-04-22T16:00:00Z" -EndTime "2015-04-22T17:00:00Z"
-    Assert-NotNull $metrics
-    Assert-True { $metrics.Count -gt 0 }
+    # **** NOTE: this section of the test has been commented out by gucalder because the current Get-AzureRmEmtric has changed, it requires different parameters and the recorded session is not longer compatible with it.
+    # $metrics = $ep1 | Get-AzureRmMetric -TimeGrain "0:5:0" -StartTime "2015-04-22T16:00:00Z" -EndTime "2015-04-22T17:00:00Z"
```

Comment:
Test section re-enabled. Sorry, I forgot to do this when I made Metric look like before.